<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageconfigEditorv4</class>
 <widget class="QWidget" name="StageconfigEditorv4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="sectionLabel">
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="sectionList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Objects</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SoundFX</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="detailsWidget">
     <widget class="QWidget" name="objectsPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="objScript"/>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="objName"/>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="3">
          <widget class="QToolButton" name="downObj">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="addObj">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="upObj">
           <property name="text">
            <string>U</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QToolButton" name="rmObj">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QListWidget" name="objList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>194</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Script</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="objType">
         <property name="text">
          <string>Object Type ID: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sfxPage">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" rowspan="5">
        <widget class="QListWidget" name="sfxList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>194</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="sfxName"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>311</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QToolButton" name="addSfx">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="upSfx">
           <property name="text">
            <string>U</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="downSfx">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="5">
          <widget class="QToolButton" name="rmSfx">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="sfxPath"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLabel" name="sfxID">
         <property name="text">
          <string>SoundFX ID: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
